tests/security/alerts/test_models.py

```
@@ -52,3 +55,19 @@ def test_alert_metadata_strips_whitespace() -> None:
 def test_alert_metadata_state_lowercased() -> None:
     md = AlertMetadata(state="  OPEN  ")
     assert md.state == "open"
+
+
+# =====================================================================
+# RuleDetails – whitespace stripping and NOT_AVAILABLE fallback
```

Ruff (0.15.15)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)